core/trino-main/src/main/java/io/trino/operator/aggregation/AggregationImplementation.java

```diff
@@ -121,7 +121,7 @@ public AggregationImplementation(
             List<AggregationParameterKind> inputParameterKinds)
     {
         this.signature = requireNonNull(signature, "signature cannot be null");
```

Comment:
I would rename all of these to be the standard "is null" since there is no benefit to the "cannot be null" phrasing